### PR TITLE
ENG-959: Update docs to encourage using explorer for creating collections

### PR DESCRIPTION
**Pull request by calummoore**

### CODE REVIEW

File: collections.mdx [hunk 1]
```diff
@@ -26,6 +26,42 @@ You can create a collection using the [Explorer](https://explorer.testnet.polyba
   and create an account.
 </Info>
 
+If you want to create your collection programmatically, see [Creating a collection with applySchema]
```
Comment (calummoore) [inline collections.mdx:29]: Can this have its own title, like  "Create collection programmatically"

File: get-started.mdx [hunk 2]
```diff
@@ -51,48 +51,16 @@ const db = polybase.Polybase({
...
-      this.id = id;
-      this.name = name;
-    }
-  }
-`,
-  "your-namespace"
-); // your-namespace is optional if you have defined a default namespace
-```
-
 For more details on creating collections, see the [collection](/collections) overview.
```
Comment (calummoore) [inline get-started.mdx:64]: Please update as I orignally asked:

If you want to create your collection programatically, see [Creating collection with applySchema](/collections#<section title you created>).

You will get the URL by previewing the change and clicking on the # next to the title

File: collections.mdx [hunk 3]
```diff
@@ -16,7 +16,45 @@ You can view our example app
 [Polybase Social](https://social.testnet.polybase.xyz) to see it working in
 action.
 
-## Create a collection
+## Create a collection programmatically
+
+If you want to create your collection programatically, see [Creating collection with applySchema](/collections#).
```
Comment (calummoore) [inline collections.mdx:21]: This should be in the get-started section, and the link should link to the title `## Create a collection programmatically`

**Comment by shaki150697** [comment]
Made the necessary changes @calummoore Let me know if this one is okay?

**Review by shaki150697 — APPROVED** [review]
Cool makes sense.

**Pull request merged**